modules/phase_field/src/userobjects/RandomEulerAngleProvider.C

```diff
@@ -0,0 +1,50 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "RandomEulerAngleProvider.h"
+
+template<>
+InputParameters validParams<RandomEulerAngleProvider>()
+{
+  InputParameters params = validParams<EulerAngleProvider>();
+  params.addClassDescription("Assign random euler angles for each grain.");
+  params.addRequiredParam<UserObjectName>("grain_tracker_object", "The FeatureFloodCount UserObject to get values from.");
+  params.addParam<unsigned int>("seed", 0, "Seed value for the random number generator");
+  return params;
+}
+
+RandomEulerAngleProvider::RandomEulerAngleProvider(const InputParameters & params) :
+    EulerAngleProvider(params),
+    _grain_tracker(getUserObject<GrainTrackerInterface>("grain_tracker_object")),
+    _angles(0)
+{
+  _random.seed(getParam<unsigned int>("seed"));
```

Comment:
Same here, you've accidentally called the static method. Make sure you pass in two integers, one for the state index and one for the seed.